challenge-api/main.ts

```diff
@@ -2,8 +2,15 @@ import mqtt from "mqtt";
 
 const BROKER_HOST = Deno.env.get("BROKER_HOST") || "localhost";
 const BROKER_PORT = Deno.env.get("BROKER_PORT") || "1883";
+const BROKER_USERNAME = Deno.env.get("BROKER_USERNAME") || "noUser";
+const BROKER_PASSWORD = Deno.env.get("BROKER_PASSWORD") || "noPassword";
```

Comment:
```suggestion
const BROKER_USERNAME = Deno.env.get("BROKER_USERNAME") || "";
const BROKER_PASSWORD = Deno.env.get("BROKER_PASSWORD") || "";
```

i think thats better if no username pwd is needed